### PR TITLE
feat(LERN-90): implemented redirect to grades page

### PR DESCRIPTION
**Pull request by kieranslost**
I kept the old version in the code incase someone wants to use it

### CODE REVIEW

**Review by deepspace1000 — CHANGES_REQUESTED** [review]
Nice 🥳

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 1  | ❗       |
    


*❗ custom/admin-landing-page*: 


| Before | After |
| --- | --- |
|  |  |

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -22,7 +21,8 @@ export function App() {
     <Wrapper>
       {showHeader && <Header />}
       <Routes>
-        <Route path={'/'} element={<LandingPage />} />
+        {/*<Route path={'/'} element={<LandingPage />} />*/}
```
[the remark this answers is not in the document]
Comment (deepspace1000) [reply frontend/src/App.tsx]: I think the LandingPage isn't use anymore is it? If not you can delete it.

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 0  | ❗       |

**Review by deepspace1000 — APPROVED** [review]

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 1  | ❗       |
    


*❗ custom/admin-landing-page*: 


| Before | After |
| --- | --- |
|  |  |

**Pull request merged**